name.abuchen.portfolio.ui/src/name/abuchen/portfolio/ui/views/taxonomy/ReBalancingViewer.java

```diff
@@ -145,12 +145,74 @@ public Color getForeground(Object element)
         });
         support.addColumn(column);
 
+        // Column which shows percentage of the target for this asset class in relationship to total assets
+        column = new Column("toBePctOfTotal", Messages.ColumnToBePctOfTotal, SWT.RIGHT, 60); //$NON-NLS-1$
+        column.setMenuLabel(Messages.ColumnToBePctOfTotal_MenuLabel);
+        column.setLabelProvider(new ColumnLabelProvider()
+        {
+            @Override
+            public String getText(Object element)
+            {
+                TaxonomyNode node = (TaxonomyNode) element;
+                // Divide to-be amount for this asset class by amount of total assets
+                // (root of asset class tree)
+                if (node.getTarget() == null)
+                    return null;
+
+                long tobe = node.getTarget().getAmount();
+                
+                long total = node.getRoot().getActual().getAmount();
```

Comment:
This calculates the percentages against *actuals* not the target. If not all assets are assigned, then this makes a difference. Will fix it.